frontend-react/src/components/ManagePublicKey/ManagePublicKey.test.tsx

```diff
@@ -73,8 +68,8 @@ describe("ManagePublicKey", () => {
                 renderApp(<ManagePublicKey />);
             });
 
-            test("renders ManagePublicKeyChooseSender", () => {
-                expect(screen.getByText(/Manage Public Key/)).toBeVisible();
+            test.skip("renders ManagePublicKeyChooseSender", () => {
```

Comment:
Skipped for now since there is an issue with re-render of ManagePublicKeyChooseSender

Reply:
Can you elaborate here?

Reply:
It looks like two instances of ManagePublicKey are getting rendered in the test so `getByText`/`getByTestId` are returning two duplicate elements.  This is likely due to the two renders:
  - https://github.com/CDCgov/prime-reportstream/pull/9007/files#diff-7cb838f0917dababbf6a240e9428c5683ccda98ebb9d01a2a423754a37e99a58R58
  - https://github.com/CDCgov/prime-reportstream/pull/9007/files#diff-7cb838f0917dababbf6a240e9428c5683ccda98ebb9d01a2a423754a37e99a58R68

We should only have one render call per example OR we should be cleaning up between test runs.  (We should be cleaning up between test runs anyway, but I mean doing it explicitly if we're going to be rendering multiple times in an example.  This is much less preferable than just having one render call per example.)